java/maven-kickstart-project/src/test/java/com/codurance/katalyst/kickstart/MockitoShould.java

```diff
@@ -0,0 +1,31 @@
+package com.codurance.katalyst.kickstart;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class MockitoShould {
+
+    public class MockedClass {
+
+        public Boolean returnFalse() {
+            return null;
+        }
+    }
```

Comment:
I think this should be an object that is within the scope of the `Calculator` context we have in the core testing phase. Obviously there aren't many dependencies you can have with a calculator, but what about introducing a class called `UnitConverter` or something which converts `1000` to `1,000` - that gives us something concrete to mock and demonstrates a little IoC?